README.md

```diff
@@ -151,27 +151,30 @@ In addition you can look at the many Discord Bots that were implemented using JD
 [And many more!](https://github.com/search?q=JDA+discord+bot&type=Repositories&utf8=%E2%9C%93)
 
 ## Download
+Latest Stable Version: [GitHub Release](https://github.com/DV8FromTheWorld/JDA/releases/latest)
```

Comment:
not a fan of advertizing releases as builds to use

Reply:
The point is to have a stable release build, dev builds can have a variance of changes that are difficult to find. Stable releases allow to see a full changelist and adapt more easily